src/pages/quero-aprender/index.js

```diff
@@ -80,6 +80,33 @@ const ContactPage = () => {
             </label>
           </div>
           <VerticalSpacing size="2x" />
+          <div>
+            <label htmlFor="document">
+              Qual é o número da sua identidade?
+              <VerticalSpacing size="1x" />
+              <Input
+                id="document"
+                name="document"
+                autoComplete="document"
+                autoCorrect="off"
+                aria-required={true}
+                enterkeyhint="send"
+                maxLength={15}
+                minLength={7}
+                mozactionhint="send"
+                placeholder="Número do seu RG"
+                title="Por favor, informe o número do seu documento de identidade"
+                type="text"
+              />
+            </label>
+            <div>
+              <small>
+                Se você for selecionado o número do seu documento será
```

Comment:
Em vez de "se você for selecionado" podemos usar "se você for uma das pessoas selecionadas" ou "se selecionarmos você". Buscamos sempre adotar linguagem neutra de gênero no site do Educodar. 

A linguagem neutra de gênero remove a generalização para o masculino, um viés que é reflexo da nossa sociedade

Reply:
Alterei essa parte para "Esta informação será utilizada para liberar sua entrada na portaria, caso você seja uma das pessoas selecionadas."